openpype/modules/muster/plugins/publish/submit_maya_muster.py

```diff
@@ -25,6 +25,7 @@ def _get_template_id(renderer):
     :rtype: int
     """
 
+    # TODO: Use settings from context?
     templates = get_system_settings()["modules"]["muster"]["templates_mapping"]
```

Comment:
From the `InstancePlugin` what would be the best way to access these settings?

However, aside of that - it even seems the function is unused? It's only used in the [plugin's `_resolve_template()` method](https://github.com/ynput/OpenPype/blob/61f7a2039b60567416d80005c046aab7a5e28de2/openpype/modules/muster/plugins/publish/submit_maya_muster.py#L209C9-L209C26) which doesn't get used within the file.

Reply:
> From the InstancePlugin what would be the best way to access these settings?

```
instance.context.data["system_settings"]
```

> However, aside of that - it even seems the function is unused? It's only used in the [plugin's _resolve_template() method](https://github.com/ynput/OpenPype/blob/61f7a2039b60567416d80005c046aab7a5e28de2/openpype/modules/muster/plugins/publish/submit_maya_muster.py#L209C9-L209C26) which doesn't get used within the file.

Maybe remove it in that case? I would be surprised if Muster would work anyway.

Reply:
Remove the unused functions, remove the todo or remove the change?